moveit_core/robot_state/src/robot_state.cpp

```diff
@@ -1957,6 +1961,14 @@ double moveit::core::RobotState::computeCartesianPath(const JointModelGroup* gro
 double moveit::core::RobotState::testJointSpaceJump(const JointModelGroup* group, std::vector<RobotStatePtr>& traj,
                                                     double jump_threshold)
 {
+  if (traj.size() < min_steps_for_jump_thresh)
+  {
+    logWarn("Truncating Cartesian path since the computed trajectory was too short to detect jumps in joint-space "
+            "Needed at least %zu steps, only got %zu. Try a lower max_step.",
+            min_steps_for_jump_thresh, traj.size());
+    traj.resize(0);
```

Comment:
i don't like that this 'truncates'. seems it should either 1) warn but return best attempt 2) auto fix the step size or 3) throw an error with this message.